constructor/preconda.py

```diff
@@ -36,9 +37,7 @@ def write_index_cache(info, dst_dir):
         write_repodata(cache_dir, url)
 
     for cache_file in os.listdir(cache_dir):
-        if cache_file.endswith(".json"):
-            files += join(cache_dir, cache_file),
```

Comment:
This `+=` pattern had no effect because `files` was already imported in `shar.py`.